.nycrc

```diff
@@ -14,6 +14,7 @@
 		"resolvers/*/test",
 		"scripts",
 		"memo-parser",
-		"lib"
+		"lib",
+		"utils"
```

Comment:
why would we want to exclude utils from coverage?

Reply:
Some of the runs were blowing up on the typescript files. I'll remove it and push and see if it still happens.

Reply:
Here's the error:

```sh
> eslint-plugin-import@2.29.1 tests-only /mnt/d/a/eslint-plugin-import/eslint-plugin-import
> npm run mocha tests/src


> eslint-plugin-import@2.29.1 mocha /mnt/d/a/eslint-plugin-import/eslint-plugin-import
> cross-env BABEL_ENV=test nyc mocha "tests/src"

/mnt/d/a/eslint-plugin-import/eslint-plugin-import/utils/declaredScope.d.ts: Unexpected token, expected ] (6:15)
```

Reply:
Here's the same error showing up in the [Appveyor pipeline](https://ci.appveyor.com/project/ljharb/eslint-plugin-import/builds/49398655/job/r9l74p4cfvb7gmat)

Reply:
i think this is maybe because of a newer version of mocha defaulting to include `.ts` files? if so, maybe we can exclude them with mocha.opts?